hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationRunnerImpl.java

```diff
@@ -89,6 +90,9 @@
     private final Address thisAddress;
     private final boolean staleReadOnMigrationEnabled;
 
+    @Probe(name = "failedBackups")
+    private final Counter failedBackupsCounter = newSwCounter();
```

Comment:
I am indecisive. it's makes it harder to interpret it, but it gives finer-grained view. wdyt?